src/odemis/driver/smartpod.py

```diff
@@ -38,11 +42,17 @@
 def add_coord(pos1, pos2):
     """
     Adds two coordinate dictionaries together and returns a new coordinate dictionary.
+
+    NOTE: pos1 and pos2 MUST contain the same axes values
+
     pos1: dict (axis name str) -> (float)
     pos2: dict (axis name str) -> (float)
     Returns ret
         dict (axis name str) -> (float)
     """
+    if pos1.keys() != pos2.keys():
```

Comment:
Now this function doesn't fit `doMoveRel` anymore... For instance, it's allowed to call `dev.moveRel({"x", 0.1})`. As the pos1 is the current position, the keys are not the same. => failure. I think the previous behaviour was fine. It should just be documented.